src/main/scala/Main.scala

```diff
@@ -1,22 +1,30 @@
 import CSVValidation.{MessageWithCsvContext, Validator}
 import com.typesafe.scalalogging.Logger
+import scopt.OParser
 
 import java.io.File
 import java.net.URI
-case class Config(inputSchema: String = "")
+case class Config(inputSchema: String = "", csvPath: String = "")
```

Comment:
I wonder whether using `Option`s here would be clearer that a value hasn't been provided?